components/style/themes/default.less

```diff
@@ -605,7 +605,7 @@
 @tabs-card-active-color: @primary-color;
 @tabs-title-font-size: @font-size-base;
 @tabs-title-font-size-lg: @font-size-lg;
-@tabs-title-font-size-sm: @font-size-base;
+@tabs-title-font-size-sm: @font-size-sm;
```

Comment:
Not modify the small tabs font size. Smaller height instead.

Reply:
I will try tomorrow.